### ISSUE TITLE
Solution for 3.7.1-2 is wrong

### ISSUE THREAD

**Issue by siche** [fool2fish/dragon-book-exercise-answers#138]

**Comment by ghost** [comment]
Yes, state D should be {0,1,2,3} with both "a" and "b" transitions mapped to D.

**Comment by MoRadwan74** [comment]
Yes indeed, but I think the mapping of "a" and "b" should remain the same "C" and "D" respectively not like what @TheProphet007 wrote.

**Comment by vskyman** [comment]
Yes， I agree with @Radwan74 .

**Comment by vskyman** [comment]
The transition table of graph 3-29 should be:

NFA State | DFA State | a | b
-- | -- | -- | --
{0} | A | B | A
{0,1} | B | C | B
{0,1,2} | C | C | D
{0,1,2,3} | D | C | D

**Comment by zihammmm** [comment]
yeah..I agree . I thought I was wrong, lol.
